Makefile

```diff
@@ -110,7 +110,7 @@ $(RELEASE_DIR):
 $(RELEASE_NOTES_DIR):
 	mkdir -p $(RELEASE_NOTES_DIR)/
 
-REGISTRY ?= ghcr.io/cluster-api-provider-k3s/cluster-api-k3s
+REGISTRY ?= ghcr.io/k3s-io/cluster-api-k3s
```

Comment:
unsure how the perms on this work for this org. Do you have any clue?

Reply:
I think we should be good as the workflow specifies write packages.

I have update the docker file to label it from this new repo. Also, i tested that the old images are still available. So this worked:

```bash
docker pull ghcr.io/cluster-api-provider-k3s/cluster-api-k3s/bootstrap-controller:v0.1.9
```